src/components/Layout/layout.js

```diff
@@ -8,6 +8,23 @@ import '../../templates/globals.css'
 
 import styles from './layout.module.css'
 
+const Favicon = () => {
```

Comment:
for the sake of modularity, since this is it's own component, it would be better to create it in a separate file called: `src/components/Layout/favicon.js` that exports this component and imports it into this file

Reply:
Moved the code to it's own component